content/en/samples/posts/2024-12-26-idle-view-app.md

```diff
@@ -0,0 +1,444 @@
+---
+title: Idle View App for webOS Devices
+date: 2024-12-26
+slug: idle-view-app
+posttype: app
+toc: true
+thumbnail: th-idle-view-app.png
+---
+
+**Author: Kangin Lee, Seungjin Kim, Tutku Genis**
+
+# Idle View App for webOS Devices
+
+
+## Overview of Project
+
+**The Personalized IdleView Application** for webOS aims to revolutionize idle screen functionality by delivering meaningful, real-time information tailored to each user. Designed with energy efficiency as main principle, this app minimizes power consumption while ensuring consistent delivery of real-time, customizable content.
+
+<p align="center"> <img src="https://github.com/gangin0221/website/blob/main/dashboard_heritage_red.png?raw=true" alt="alt text" /> </p>
+
+<p align="center">
+  <img src="https://github.com/gangin0221/website/blob/main/brightness_control_settings.png?raw=true" alt="Image 1" width="49%" />
+  <img src="https://github.com/gangin0221/website/blob/main/background_change_widget.png?raw=true" alt="Image 2" width="49%" />
+</p>
+                                                                            
+
+
+### Key Features
+This section highlights the core functionalities of the IdleView project.
+
+- **Customizable Dashboards and Widgets**  
+  Users can personalize their IdleView screen with widgets for weather updates, calendar schedules, media playback, and more.
+  <p align="center"> <img src="https://github.com/gangin0221/website/blob/main/dashboard_heritage_red.png?raw=true" alt="Customizable Dashboard" width="100%" /> </p> <p align="center"> <small>*Example of a customizable dashboard with various widgets.* </p>
```

Comment:
please replace it with a different image, as the same one is currently being used.